app/src/main/java/mozilla/lockbox/view/FxALoginFragment.kt

```diff
@@ -28,6 +29,18 @@ import mozilla.lockbox.support.isDebug
 
 class FxALoginFragment : BackableFragment(), FxALoginView {
     override var webViewRedirect: ((url: Uri?) -> Boolean) = { _ -> false }
+//    private var errorHelper = NetworkErrorHelper()
+//
+//    override val networkErrorVisibility: Consumer<in Boolean>
```

Comment:
The pattern we've used elsewhere is to have these methods be called on the fragment, rather than using a `Consumer`.

Reply:
Right now, the presenters kick off `dispatcher.dispatch(NetworkAction.CheckConnectivity)` and the `Fragments` are looking at the `NetworkStore` to see what the current state is (consuming this value from the network store)